### ISSUE TITLE
[Roadmap] PyTorch Geometric 1.6.1 and 1.7.0

### ISSUE THREAD

**Issue by rusty1s** [rusty1s/pytorch_geometric#1465]
## Roadmap: PyTorch Geometric 1.6.1 and 1.7.0

**Comment by sbonner0** [comment]
I am not sure if this is helpful, or indeed even the correct place for this - please let me know if not and I can remove it. However I thought it might be useful to note that I have been using pytorch geometric with pytorch 1.6.0 over the past few days and haven't noticed any issues. My exact versions of the relevant packages are:
`torch                      1.6.0rc7
torch-cluster              1.5.6
torch-geometric            1.6.0
torch-scatter              2.0.5
torch-sparse               0.6.6
torch-spline-conv          1.2.0
`
I can run the test suite if it would be helpful?

**Comment by rusty1s** [comment]
Thanks for letting me know. I also encountered no major issues while upgrading. However, i still need to provide updates wheels for the latest PyTorch release.

**Comment by yangky11** [comment]
Hi @rusty1s ,

I tried to install PyTorch Geometric with PyTorch 1.6.0 by following the instructions [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html). However, I cannot find the updated wheels for torch-sparse and torch-cluster. May I know if they will become available in the near future? Thanks!

**Comment by rusty1s** [comment]
Sure, they should be available next week :)

**Comment by aluo-x** [comment]
Just built sparse/scatter/cluster/spline-conv/geometric with PyTorch master, CUDA 11, cudnn8, GCC 9.3, MAGMA 2.5.2. Encountered no issues either compiling or running.

**Comment by lgray** [comment]
Just to add more to this, I've been using torch 1.6 throughout its development period, including the RCs. Never any problems.

**Comment by GazelleThr** [comment]
Hi there,

I sm trying to read my data using TORCH_GEOMETRIC.IO.TU code. My data type is exactly similar to the enzymes dataset. However, the program stops with the Illegal instruction (core dumped) error at this line "node_slice = torch.cumsum(torch.from_numpy(np.bincount(batch)), 0)" in the split function. It seems that the problem is with the dimension but I can't figure out what the solution is. Could you please help with this. 

Please note, I installed the package exactly as it is mentioned in the document.

Thanks

**Comment by GazelleThr** [comment]
Hi Matthias,

Thanks for your quick reply. I have created a new issue here:
https://github.com/rusty1s/pytorch_geometric/issues/1601

**Comment by jannessm** [comment]
Hi Matthias,
maybe it would be worth to replace network-x by cugraph, when gpu is supported.

I just have heard about it and thought, that it would make sense to give you a hint, if you havent heard about it before.

Regrads.

**Comment by rusty1s** [comment]
cugraph conversions is a good idea, I added it to the roadmap. The SuperpixelDataset is already fixed in master.

**Comment by rusty1s** [comment]
**Just released [PyTorch Geometric 1.6.2](https://github.com/rusty1s/pytorch_geometric/releases/tag/1.6.2)!**

Thanks to many wonderful contributors :) Please let me know if you encounter any issues.

**Comment by eduardohenriquearnold** [comment]
Can we add the FAConv to the next release? It's already in the source. Thanks!

**Comment by rusty1s** [comment]
I have plans to release a new version today :)

**Comment by karthi0804** [comment]
@rusty1s Very eager to get half support. When can we expect this? Thanks in advance.

**Comment by rusty1s** [comment]
I'm pretty sure that PyTorch 1.9.0 contains all the necessary tools to allow for half-precision in `torch-scatter` and `torch-sparse`. I will look into it.

**Comment by always-ready2learn** [comment]
Hi @rusty1s I have changes TGN.py model for link prediction into edge classification. The model runs for me but unfortunately loss is very high and remains constant through out which means that model is not learning at all. I am not sure what I am missing here. So far I am using similar TGN link prediction example and have made following changes in the train and test functions as:         
       y_pred = pos_out.argmax(dim=1)
        y_true = batch.y
        loss = criterion(y_pred.float(), y_true.float())
I am not sure what I am missing here. the loss value turns out to be 8.02 and remains constant, same goes for accuracy which turns out to be a constant value of .65.Could you think of what I might be missing please?

**Comment by rusty1s** [comment]
If you want to perform classification, then I think `y_true` shouldn't be a floating-point tensor. What's the loss function you are using? Note that this is best discussed in a separate issue/discussion - not necessarily in this roadmap issue here :)

**Comment by always-ready2learn** [comment]
Hi thanks for your reply, I changed y_true to long and y_pred to float. The loss function I tried with is BCELoss and CrossEntropyLoss. Same issue is occurring with both. 
Thanks for pointing that, I found it to be similar topic but you are right issue is quite diff. I’ll move this discussion to a separate issue sooner. :)

**Comment by rusty1s** [comment]
I think this is no longer an issue since we are using PyTorch's `spspmm` functionality by now.
